apps/portals/src/Navbar.tsx

```diff
@@ -260,7 +260,11 @@ function Navbar() {
             isSynapseSubdomainOrLocal && ( // desktop version, show dropdown
               <>
                 <Dropdown className="user-loggedIn">
-                  <Dropdown.Toggle variant="light" id="user-menu-button">
+                  <Dropdown.Toggle
+                    variant="light"
+                    id="user-menu-button"
+                    aria-label="User Dropdown Menu"
```

Comment:
Couple of small changes for https://sagebionetworks.jira.com/browse/PORTALS-2688